src/main/java/duke/command/UnmarkCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.TaskList;
+
+public class UnmarkCommand extends Command {
+
+    private int index;
+
+    public UnmarkCommand(int command) {
+        index = command;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        try {
+            tasks.unmark(index);
+            storage.unmark(index);
+            ui.showMessage("Got it! Task " + index + " has not yet been completed:\n  " + tasks.showTask(index));
+        } catch (IndexOutOfBoundsException e) {
+            ui.showError("You've given me an invalid task to unmark!");
+        }
+    }
+
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+}
```

Comment:
You can consider combining MarkCommand and UnmarkCommand into a single file with an isDone boolean to represent the state!